easybuild/easyconfigs/l/libcerf/libcerf-2.1-GCCcore-11.3.0.eb

```
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'libcerf'
+version = '2.1'
+
+homepage = 'https://jugit.fz-juelich.de/mlz/libcerf'
+
+description = """
+ libcerf is a self-contained numeric library that provides an efficient and
+ accurate implementation of complex error functions, along with Dawson,
+ Faddeeva, and Voigt functions.
+"""
+
+toolchain = {'name': 'GCCcore', 'version': '11.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://jugit.fz-juelich.de/mlz/libcerf/-/archive/v%(version)s/']
+sources = ['libcerf-v%(version)s.tar.gz']
+checksums = ['8a1cd8b7fae04b82a95168252129b8c1baca098a285ff8d3f25781dead14b75a']
+
+builddependencies = [
+    ('binutils', '2.38'),
+    ('CMake', '3.23.1'),
+    ('Perl', '5.34.1'),  # required for pod2html
```

the `-minimal` version should suffice here, it provides `pod2html`
```suggestion
    ('Perl', '5.34.1', '-minimal'),  # required for pod2html
```